### ISSUE TITLE
Accessibility: MAS40B: Inspect: IsTogglePatternAvailable pattern is true for the tool bar items in "Variable explorer" window

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/RTVS#3342]
**Platform:**
Windows 10 RS1
Visual Studio Enterprise 2017 Preview
Version: 15.0.26340.0

**Repro Steps:**
1. Launch Visual Studio
2. Hit "Alt+R", to open "Rtools" menu
3. Using arrow keys navigate to "Data Science Settings" menu item and hit enter
4. In the "R Interactive" pane, enter the command for ex., "x<-iris" and hit enter to get the variable in the "Variable Explorer"
5. Navigate to "Variable Explorer" pane, hit "Shift+Alt" key to move to the tool bar
6. Tab "Import Dataset Into R Session From Web URL", "Import Dataset Into R Session From Text File" and "Delete All Variables" buttons in the tool bar and observe the properties in inspect

**Expected Result:**
IsTogglePatternAvailable property should be false

**Actual Result:**
IsTogglePatternAvailable property is true

**Narrator Behavior:**
Narrator announces as "off, Import Dataset Into R Session From Web URL..., button 
Import Dataset Into R Session From Web URL"

**User Impact:**
Screen reader users will misunderstand that the button can be toggled and there is change in state if IsTogglePattern is set to true. Also the screen reader users will get annoyed and may get redundant information if name and help text are similar

![inspect_istogglepattern_tool bar buttons](https://cloud.githubusercontent.com/assets/22956130/24296070/8af4a924-10c3-11e7-814f-f1722f8b15a1.jpg)
![narrator log_buttons in the tool bar](https://cloud.githubusercontent.com/assets/22956130/24296069/8af4862e-10c3-11e7-9cf3-b5b807b74f43.jpg)

**Comment by MikhailArkhipov** [comment]
The VSCT definition looks standard and no different from other buttons.

**Comment by MikhailArkhipov** [comment]
If this happens to other buttons as well we should just send it to IDE

**Comment by AlexanderSher** [comment]
yes, it is the same for all toolbar buttons in the Team Explorer and Solution Explorer.

**Comment by MikhailArkhipov** [comment]
VSTS 401433
